sources/functions/curl

```diff
@@ -56,7 +56,7 @@ build_curl() {
     unzip -q -o /tmp/curl.zip -d /tmp >> $log 2>&1
     rm /tmp/curl.zip
 
-    cd /tmp/curl-debian-bullseye-backports
+    cd /tmp/curl-debian-bookworm-backports
     rm CMakeCache.txt >> $log 2>&1
```

Comment:
should be using `rm_if_exists`

Reply:
Problem resolved. Utils is already imported for c-ares. Imported for curl as well.